examples/wallet.example.ts

```diff
@@ -86,4 +90,29 @@ const hethers = require("ethers");
 	console.log(`Decrypted Wallet Sync:`);
 	console.log(decryptedWalletSync);
 
+	const wallet = hethers.Wallet.createRandom();
+	const txData = await new TransferTransaction()
+		.setTransactionId(TransactionId.generate("0.0.98"))
+		.setNodeAccountIds([new AccountId(0, 0, 3)])
+		.freeze()
+		.toBytesAsync();
+	const tx = {
+		to : getAddressFromAccount("0.0.98"),
+		from: "",
+		gasLimit: 0,
+		data: hexlify(txData),
```

Comment:
this should not be the `data` property. Data property is Ethereum is `{functionSignature+arguments}`